### PR TITLE
Make anchors more upstream/downstream friendly

### PR DESCRIPTION
**Pull request by maximiliankolb**
This PR consists of three commits to make anchors more upstream and downstream friendly. The work with anchors is not done (by far), but this is a start.

### CODE REVIEW

**Review by ehelms — APPROVED** [review]
Will let @melcorr or @lzap take a look to make sure I didn't miss some nuance

File: guides/doc-Planning_Guide/topics/Introduction.adoc [hunk 1]
```diff
@@ -27,21 +27,21 @@ ifndef::satellite[]
 include::../common/modules/snip_red-hat-images.adoc[]
 endif::[]
 
-[[figu-Red_Hat_Satellite-Architecture_Guide-Red_Hat_Satellite_6_System_Architecture-Red_Hat_Satellite_6_System_Architecture]]
+[[figu-Architecture_System_Architecture-Red_Hat_Satellite_6_System_Architecture]]
```
Comment (melcorr) [inline guides/doc-Planning_Guide/topics/Introduction.adoc:30]:
```suggestion
[[figu-Architecture_System_Architecture-System_Architecture]]
```
We could remove further?

**Review by melcorr — COMMENTED** [review]
One small suggestion.

File: guides/doc-Planning_Guide/topics/Introduction.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/doc-Planning_Guide/topics/Introduction.adoc]: Removed (two occurrences) & based to master.

**Review by melcorr — APPROVED** [review]
LGTM

**Pull request merged**